sdk/eventhub/Azure.Messaging.EventHubs/tests/EventProcessorClient/MockCheckPointStorageTests.cs

```diff
@@ -7,40 +7,41 @@
 using System.Threading.Tasks;
 using Azure.Messaging.EventHubs.Processor;
 using NUnit.Framework;
+using Azure.Messaging.EventHubs.Samples.Infrastructure;
```

Comment:
Alphabetical order here as well, but `System.Threading.Tasks` should be kept at the top.